isofit/utils/segment.py

```diff
@@ -30,7 +30,7 @@
 from isofit.core.common import envi_header, ray_initiate
 
 
-@ray.remote
+@ray.remote(num_cpus=1)
```

Comment:
With the addition of `ray.shutdown(); time.sleep()` in the test case, is this still necessary? Has it been tested if this impacts performance in any way?

Reply:
@jammont I haven't tested if this impacts performance as the segmentation is generally pretty fast and not a bottleneck. You're probably right that this keyword is not needed as only the addition of `ray.shutdown(); time.sleep()` solved the issue. I'll remove it and see if the workflows still pass.

Reply:
I think #472 might actually handle this, but ti doesn't hurt.  I've tested for performance, and this does not hurt us.

Reply:
Discussed verbally at the team meeting: we'll be implementing this to all ray jobs in a future PR as we do not want any threading.